openedx.yaml

```diff
@@ -0,0 +1,9 @@
+# This file describes this Open edX repo, as described in OEP-2:
+# http://open-edx-proposals.readthedocs.io/en/latest/oeps/oep-0002.html#specification
+
+nick: adap
```

Comment:
This is used by the open source team for some reports, so the goal is to have it short. You could talk to @nedbat about whether it's still used.